src/main/java/org/folio/rest/impl/PermsAPI.java

```diff
@@ -857,12 +851,8 @@ public void getPermsPermissionsById(String id, Map<String, String> okapiHeaders,
                                       Context vertxContext) {
     try {
       String tenantId = TenantTool.tenantId(okapiHeaders);
-      Criteria idCrit = new Criteria();
-      idCrit.addField(ID_FIELD);
-      idCrit.setOperation("=");
-      idCrit.setVal(id);
       PostgresClient.getInstance(vertxContext.owner(), tenantId)
-          .get(TABLE_NAME_PERMS, Permission.class, new Criterion(idCrit), true, false, getReply -> {
+          .get(TABLE_NAME_PERMS, Permission.class, getIdCriterion(id), true, false, getReply -> {
```

Comment:
Can PgUtil.getById been used?

Reply:
Actually, yes. But changes behavior a bit (not for the worse)